src/pet.js

```diff
@@ -0,0 +1,59 @@
+const MAXIMUM_FITNESS = 10;
+const AGE_AT_BIRTH = 0;
+const NOT_HUNGRY = 0;
+const isDeadError = new Error('Sorry, your pet has gone to live on the big farm in the sky');
```

Comment:
like that you've taken this out as it's own variable, as you using this many times throughout your code